src/main/webapp/file.xhtml

```diff
@@ -309,11 +309,11 @@
                                                     #{bundle['metrics.file.title']}
                                                     <ui:fragment rendered="#{!settingsWrapper.makeDataCountDisplayEnabled}">
                                                         <span class="glyphicon glyphicon-question-sign tooltip-icon" data-toggle="tooltip" data-placement="auto top" 
-                                                            data-trigger="hover" data-original-title="#{bundle['metrics.dataset.tip.default']}"></span>
+                                                            data-trigger="hover" data-original-title="#{bundle['metrics.file.tip.default']}"></span>
```

Comment:
This pull request changes...

metrics.dataset.tip.default=Aggregated metrics for this dataset.

... to this...

metrics.file.tip.default=Metrics for this individual file.